ui/pages/onboarding-flow/secure-your-wallet/secure-your-wallet.js

```diff
@@ -47,6 +47,17 @@ export default function SecureYourWallet() {
 
   const trackEvent = useContext(MetaMetricsContext);
 
+  const handleOnShowSrpDetailsModal = useCallback(() => {
+    trackEvent({
+      category: MetaMetricsEventCategory.Onboarding,
+      event: MetaMetricsEventName.SrpDefinitionClicked,
+      properties: {
+        location: 'secure_your_wallet',
```

Comment:
Add all locations as constants. if possible, use route names
@nakulmk pls check if these names can match route names.
Is the pr on segment schema merged first?